.travis.yml

```diff
@@ -10,6 +10,7 @@ services: mongodb
 before_install:
   - if [[ $(phpenv version-name) = "7.1" ]]; then pecl install mongodb; fi
   - echo "extension = mongodb.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+  - phpenv config-rm xdebug.ini || true
```

Comment:
why? without this we will have no coverage metrics

![image](https://user-images.githubusercontent.com/6104262/49454483-62561d80-f7cc-11e8-9c94-7588811c89ca.png)